src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs

```diff
@@ -1120,6 +1118,47 @@ private Task<GetAccountTransferConnectionsResponse> GetTransferConnectionsNoAuth
                     }, model.HashedAccountId, externalUser);
         }
 
+        public async Task<OrchestratorResponse<TransferCommitmentViewModel>> GetCommitmentDetailsForTransfer(
+            string hashedTransferAccountId, string hashedCommitmentId, string externalUserId)
+        {
+            var accountId = _hashingService.DecodeValue(hashedTransferAccountId);
+            var commitmentId = _hashingService.DecodeValue(hashedCommitmentId);
+            _logger.Info($"Getting Commitment Details, Transfer Account: {accountId}, CommitmentId: {commitmentId}");
+
+            return await CheckUserAuthorization(async () =>
+            {
+                var data = await _mediator.SendAsync(new GetCommitmentQueryRequest
+                {
+                    AccountId = accountId,
+                    CommitmentId = commitmentId,
+                    CallType = CallType.TransferSender
+                });
+
+                var apprenticeships = data.Commitment.Apprenticeships ?? new List<Apprenticeship>();
```

Comment:
It might be worth popping some of this code into a new TransfersMapper to handle the creation of this VM.

Reply:
I'm doing this in branch 1128, as I won't have to merge it again